public/pages/DetectorsList/containers/List/List.tsx

```diff
@@ -458,16 +585,15 @@ export const DetectorList = (props: ListProps) => {
             <ListActions
               onStartDetectors={handleStartDetectorsAction}
               onStopDetectors={handleStopDetectorsAction}
-              //onDeleteDetectors={handleDeleteDetectorsAction}
-              detectors={selectedDetectorsForAction}
+              onDeleteDetectors={handleDeleteDetectorsAction}
               isActionsDisabled={selectedDetectorsForAction.length === 0}
             />,
             <EuiButton fill href={`${PLUGIN_NAME}#${APP_PATH.CREATE_DETECTOR}`}>
               Create detector
             </EuiButton>,
           ]}
         >
-          {showConfirmModal ? confirmModal : null}
+          {confirmModal ? confirmModal : null}
```

Comment:
what if just put `confirmModal` there? looks like its default value is null.

Reply:
Yes, good catch. will change